### DISCUSSION TITLE
Github Education access request denied

### DISCUSSION THREAD

**Discussion by ilo80** [community/community#69530] [GitHub Education]
### Hi everyone,

I have tried to join the GitHub Education program several times (21 times) but all my applications have been rejected despite the use of a school certificate and my school ID. Being French, I also provided an English translation of my documents for registering in the program.

I received several errors such as :
> Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.

Or

> Please correct your GitHub billing information with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.

Any idea how I can get this solved?

Thanks in advance.

(Sorry for my bad english level)

**Comment by andriivitiv** [comment]
Had the same error. Fixed it by writing my name and school name on a paper in English and filling out the billing information also in English.

**Reply by ilo80** [reply]
Thanks you for the answer. I have tried this but got the following error :
> Please upload proof of your current school affiliation
